sources/Application/Instruments/SampleInstrumentDatas.h

```diff
@@ -1,6 +1,6 @@
 #define SEMITONE_FREQ_INTERVAL 1.0594630943592952645618252949461F
 
-const char *loopTypes[SILM_LAST] = {"none", "loop", "oscillator",
+const char *loopTypes[SILM_LAST] = {"none", "loop", "pingpong", "oscillator",
```

Comment:
maybe we should name this "pingpong *EXP*" or something alike. Does it fit?

Reply:
Actually, I'll take this back, since it would be a problem changing names on saved projects after the fact.